src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -21,7 +23,7 @@
 class HeaderBag implements \IteratorAggregate, \Countable
 {
     protected $headers = array();
-    protected $cacheControl = array();
```

Comment:
and btw, you now need to clone the CacheControl when cloning the HeaderBag instead of keeping the same one in cloned header bags, otherwise it is also a BC break (HeaderBag is cloneable currently)